src/utils/queue_builder.rs

```diff
@@ -26,6 +29,10 @@ pub fn queue_builder(
     let mut status = BuilderState::Fresh;
 
     loop {
+        if let Err(e) = remove_tempdirs() {
+            report_error(&anyhow::anyhow!(e).context("failed to remove temporary directories"));
+        }
```

Comment:
This is probably more frequent than necessary, it only needs to cleanup on startup. But this doesn't hurt either.

Reply:
I'm fine with this, it feels more like a cleanup for a single build.

Reply:
where are the tempfiles created? wouldn't the better place for the cleanup the place where the mess is made? ( probably `RustwideBuilder::build_package`)

Reply:
( then it would also be easy to add a small test to the build-tests that covers this :) )

Reply:
Hmm, I feel nervous about hard-coding the assumption that only one build is going at once into `rustwide_builder`. It seems easy to overlook whenever we get multiple builders set up.